Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.8-slim-buster
+WORKDIR /code
+COPY . /code/
+RUN pip3 install -r requirements.txt
+ENTRYPOINT [ "/code/scripts/start.sh" ]
+CMD [ "python","/code/app/main.py"]
```

Comment:
Either the start.sh or main.py is needed. Not both.

Reply:
Fixed on (82d5b9ff2c4c5ed3c488d4b4d8ee176c32769f4c)